frontend/src/mocks/data/search.ts

```diff
@@ -0,0 +1,88 @@
+import { stations } from '@mocks/data/stations';
+
+import { QUICK_CHARGER_CAPACITY_THRESHOLD } from '@constants/chargers';
+import { MAX_SEARCH_RESULTS } from '@constants/stationSearch';
+
+export const getSearchedStations = (searchWord: string) => {
+  const searchApiStations = stations.map((station) => {
+    const { stationId, stationName, chargers, address, latitude, longitude } = station;
+
+    const onlyCapacity = chargers.map(({ capacity }) => capacity);
+    const speed = onlyCapacity.map((num) =>
+      num >= QUICK_CHARGER_CAPACITY_THRESHOLD ? 'QUICK' : 'STANDARD'
+    );
```

Comment:
다른 곳에서 onlyCapacity를 안 쓰고 있는 것 같아서 이건 어떤가요
```suggestion
  const speed = chargers.map(({ capacity }) => capacity >= 
    QUICK_CHARGER_CAPACITY_THRESHOLD ? 'QUICK' : 'STANDARD'
  );
```

Reply:
오 맵핑을 한번 더 줄일 수 있었군요

Reply:
반영했습니다